src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -141,91 +144,95 @@ public Response getID(@PathParam("id") int id, @PathParam("updateInterval") int
-        };
-        return Response.ok(feedList).build();
-    }
-
-    // Return Log information about the feed with {id}
-    @GET
-    @Path("/{id : \\d+}/log/{toggledData: .*}/pagination/{currentPage: \\d+}/{rowsPerPage: \\d+}")
-    @Produces(MediaType.APPLICATION_JSON)
-    public Response getRtFeedLogDetails(
-            @PathParam("id") int id,
-            @PathParam("toggledData") String hideErrors,
-            @PathParam("currentPage") int currentPage,
-            @PathParam("rowsPerPage") int rowsPerPage) {
+        mergeMonitorData.setViewErrorSummaryModelList(feedSummary);
 
         List<ViewErrorLogModel> feedLog;
         String [] removeIds = hideErrors.split(",");
 
         // Getting the value of fromRow from the rowsPerPage and currentPage values.
-        int fromRow = (currentPage - 1) * rowsPerPage;
-
-        Session session = GTFSDB.InitSessionBeginTrans();
+        fromRow = (logCurPage - 1) * logRowsPerPage;
         feedLog = session.createNamedQuery("ErrorLogByrtfeedID", ViewErrorLogModel.class)
-                .setParameter(0, id)
-                .setParameter(1, id)
+                .setParameter(0, gtfsRtId)
+                .setParameter(1, gtfsRtId)
+                .setParameter(2, currentTimestamp)
                 .setParameterList("errorIds", removeIds)
                 .setFirstResult(fromRow)
-                .setMaxResults(rowsPerPage)
+                .setMaxResults(logRowsPerPage)
                 .list();
-        GTFSDB.commitAndCloseSession(session);
+
         for (ViewErrorLogModel viewErrorLogModel: feedLog) {
-            String formattedTimestamp = getDateFormat(viewErrorLogModel.getOccurrence(), id);
+            String formattedTimestamp = getDateFormat(viewErrorLogModel.getOccurrence(), gtfsRtId);
             viewErrorLogModel.setFormattedTimestamp(formattedTimestamp);
+            viewErrorLogModel.setOccurrence(TimeUnit.MILLISECONDS.toSeconds(viewErrorLogModel.getOccurrence()));
             viewErrorLogModel.setTimeZone(agencyTimezone);
         }
-        GenericEntity<List<ViewErrorLogModel>> feedList = new GenericEntity<List<ViewErrorLogModel>>(feedLog) {
-        };
-        return Response.ok(feedList).build();
-    }
+        mergeMonitorData.setViewErrorLogModelList(feedLog);
 
-    // Returns number of iterations a feed has gone through
-    @GET
-    @Path("/{id : \\d+}/feedIterations")
-    @Produces(MediaType.APPLICATION_JSON)
-    public Response getFeedIterationsCount(@PathParam("id") int id) {
-        ViewFeedIterationsCount iterationsCount;
-        Session session = GTFSDB.InitSessionBeginTrans();
-        iterationsCount = (ViewFeedIterationsCount) session.createNamedQuery("feedIterationsCount", ViewFeedIterationsCount.class)
-                            .setParameter(0, id).uniqueResult();
-        GTFSDB.commitAndCloseSession(session);
+        GTFSDB.closeSession(session);
```

Comment:
I assume the reason you switched from ` GTFSDB.commitAndCloseSession(session);` to just `GTFSDB.closeSession(session);` is because this is a READ operation that doesn't write any data to the server?  Were there performance issues you saw when committing the session?  Or was there another reason why you used a new method that doesn't include a commit?